0.4.9->0.4.10 breaking change: no method named `to_rfc3339_opts` for type `DateTime<Utc>`

```rust
fn main() {
    use chrono::{DateTime, Utc, NaiveDateTime, SecondsFormat};
    let dt = DateTime::<Utc>::from_utc(NaiveDateTime::from_timestamp(42, 0), Utc);
    dbg!(dt.to_rfc3339_opts(SecondsFormat::Secs, true));
}
```
This program compiles without errors with `chrono = { version = "=0.4.9", default-features = false }`, but with `chrono = { version = "=0.4.10", default-features = false }` there is an error:
```
error[E0599]: no method named `to_rfc3339_opts` found for type `chrono::datetime::DateTime<chrono::offset::utc::Utc>` in the current scope
 --> src\main.rs:4:13
4 |     dbg!(dt.to_rfc3339_opts(SecondsFormat::Secs, true));
  |             ^^^^^^^^^^^^^^^ method not found in `chrono::datetime::DateTime<chrono::offset::utc::Utc>`
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -10,6 +10,14 @@ Versions with only mechanical changes will be omitted from the following list.
 
 ## 0.4.10
 
+### Compatibility notes
+
+* Putting some functionality behind an `alloc` feature to improve no-std
+  support (in #341) means that if you were relying on chrono with
+  `no-default-features` *and* using any of the functions that require alloc
+  support (i.e. any of the string-generating functions like `to_rfc3339`) you
+  will need to add the `alloc` feature in your Cargo.toml.
+
 ### Improvements
 
 * `DateTime::parse_from_str` is more than 2x faster in some cases. (@michalsrb

